### ISSUE TITLE
Option to not include Apex file meta information

### ISSUE THREAD

**Issue by IlyaMatsuev** [cesarParra/apexdocs#72]
Hi,
Thanks for creating this CLI, it helps a lot. However, I have a suggestion about what's being printed in the .md file for an Apex class.
Doesn't matter what options and generators I use, I always get ApexClass meta information in the final markdown file. I don't really see much sense in printing this information as I've never seen in that somebody would expose and not delete an Apex class with `Status: Inactive`.  ApexClass API version is a bit more important but again, most of the time it's not required for a user to know the exact API version.

So, would it be possible to have at least a CLI option to disable that ApexClass meta information to be printed?

What I'm running:
```sh
npx apexdocs-generate -s ./src/main -t ./docs -p global -g docsify --defaultGroupName Classes
```

Example source file:
```java
/**
 * @description Some description
 */
global abstract class Service implements IService {}
```

Expected result .md file:
```md
# Service

Some description


**Implemented types**

[IService](/Interfaces/IService.md)
```

Actual result .md file:
```md
# Service

`APIVERSION: 55`

`STATUS: ACTIVE`

Some description


**Implemented types**

[IService](/Interfaces/IService.md)
```

**Comment by IlyaMatsuev** [comment]
Cool, thanks for the quick feedback

**Comment by cesarParra** [comment]
Available in 2.14.0 https://www.npmjs.com/package/@cparra/apexdocs/v/2.14.0. Let me know if anything could be improved or is not working
